### PR TITLE
Event Section (WIP)

### PR DESCRIPTION
**Pull request by rabellamy**
See issue #46

### CODE REVIEW

**Comment by misscs** [comment]
@rabellamy Great start here. We need an example of meetup json return to finalize the format of the json structure which may have an effect on the event teaser and node page

**Comment by rabellamy** [comment]
@misscs your feedback please!!!!!

File: events/_event.jade [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
+for event, slug in public.events.event._data
+
+  if current.source === slug
+
+    h1 #{event.name}
+    time.dt-start(datetime="2013-06-30 12:00")
+      sup #{event.start_date}
+    time.dt-end(datetime="2013-06-30 18:00")
```
Comment (misscs) [inline events/_event.jade]: Make sure `datetime` is set to `event.start_date` or `event.end_date`

File: events/_event.jade [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    .sponsors
+      h4 Sponsors
+      each sponsor, delta in event.sponsors
+        .sponsor
+          span.sponsor-name: a(href="#{event.sponsors[delta].url}")= event.sponsors[delta].name
+
+
+
+
+
```
Comment (misscs) [inline events/_event.jade]: Will want to use `h-event` classes

**Comment by misscs** [comment]
Closing and reopening this PR on the `master` branch.

**Pull request closed without merging**